src/merkle/mmr/partial.rs

```diff
@@ -2,54 +2,56 @@ use super::{MmrDelta, MmrProof, Rpo256, RpoDigest};
 use crate::{
     merkle::{
         mmr::{leaf_to_corresponding_tree, nodes_in_forest},
-        InOrderIndex, MerklePath, MmrError, MmrPeaks,
+        InOrderIndex, InnerNodeInfo, MerklePath, MmrError, MmrPeaks,
     },
     utils::collections::{BTreeMap, Vec},
 };
 
-/// Partially materialized [Mmr], used to efficiently store and update the authentication paths for
-/// a subset of the elements in a full [Mmr].
+// PARTIAL MERKLE MOUNTAIN RANGE
+// ================================================================================================
+/// Partially materialized Merkle Mountain Range (MMR), used to efficiently store and update the
+/// authentication paths for a subset of the elements in a full MMR.
 ///
 /// This structure store only the authentication path for a value, the value itself is stored
 /// separately.
 #[derive(Debug)]
 pub struct PartialMmr {
-    /// The version of the [Mmr].
+    /// The version of the MMR.
```

Comment:
It would be nice to fix the links instead of removing them. Documentation is easier to use when these links work.